pallets/proposals/src/impls.rs

```diff
@@ -439,7 +440,33 @@ impl<T: Config> Pallet<T> {
             now,
         ));
 
+        //once the voting is complete check if the milestone is eligible for auto approval
+        //Getting the total threshold required for the milestone to be approved based on the raised funds
+        let funding_threshold: BalanceOf<T> = project.raised_funds * T::PercentRequiredForVoteToPass::get().into()/100u32.into();
+
+        let mut milestone = project.milestones.get_mut(&milestone_key).unwrap().clone();
+        //if the yay votes are both greater than the nay votes and the funding threshold then the milestone is approved
+        if (updated_vote.yay >= updated_vote.nay) && (updated_vote.yay >= funding_threshold) {
```

Comment:
Arguably only check if the threshhold has to be met even if its 40% and nay > yay.

Reply:
Ok so then it means that even if the nayers are more the milestones could be approved because of the less yayers who has more contribution?